eks-anywhere-common/Addons/Partner/dynatrace/dynatrace-variables.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: dynatrace-variables
+  namespace: flux-system
+data:
+  # api url for dynatrace environment 
+  apiurl: "https://sometenant.live.dynatrace.com/api"
```

Comment:
is this a correct URL?

Reply:
changed contents of this file to include real values for the dynatrace environment we configured for this purpose